content/apt/apt-1/contents.lr

```diff
@@ -58,11 +58,25 @@ Warning symptom, when running sudo apt update:
    Skipping acquire of configured file 'main/binary-i386/Packages' as repository 'http://deb.torproject.org/torproject.org focal InRelease' doesn't support architecture 'i386'
    ```
  
-#### 3. Then add the gpg key used to sign the packages by running the following commands at your command prompt
+#### 3. Then import the gpg key used to sign the packages
+   
+Download the key directly into `/etc/apt/trusted.gpg.d` using cURL:
+
+   ```
+   sudo curl https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc -o /etc/apt/trusted.gpg.d/torproject.asc
+   ```
+   
+Or download the key directly into `/etc/apt/trusted.gpg.d` using Wget:  
+
+   ```
+   sudo wget https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc -O /etc/apt/trusted.gpg.d/torproject.asc
+   ```
+   
+Or receive the key from a keyserver using gpg, then export the key into `/etc/apt/trusted.gpg.d` directory:
 
    ```
-   # wget -qO- https://deb.torproject.org/torproject.org/A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89.asc | gpg --import
-   # gpg --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | apt-key add -
+   gpg --keyserver=hkp://keys.gnupg.net --receive-keys 74A941BA219EC810
```

Comment:
deb.torproject.org signing key was trashed during an attack: 
https://lists.torproject.org/pipermail/tor-project/2019-June/002378.html

And we don't recommend in our official docs to download it from public key servers.